aries_vcx/src/handlers/issuance/credential_def/mod.rs

```diff
@@ -154,25 +113,27 @@ fn _try_get_cred_def_from_ledger(issuer_did: &str, cred_def_id: &str) -> VcxResu
 impl CredentialDef {
     pub fn create_and_store(source_id: String, config: CredentialDefConfig, revocation_details: RevocationDetails) -> VcxResult<Self> {
         trace!("CredentialDef::create_and_store >>> source_id: {}, config: {:?}, revocation_details: {:?}", source_id, config, revocation_details);
+        let CredentialDefConfig { issuer_did, schema_id, tag } = config;
+        let (_, schema_json) = anoncreds::get_schema_json(&schema_id)?;
+        let (cred_def_id, cred_def_json) = anoncreds::generate_cred_def(&issuer_did,
+                                                                        &schema_json,
+                                                                        &tag,
+                                                                        None,
+                                                                        revocation_details.support_revocation.clone())?;
+
+        let rev_reg = if revocation_details.support_revocation.unwrap() {
```

Comment:
This panics if `support_revocation` is `None`, which needs to be avoided. The original implementation did that.

Reply:
fixed, defaults to `false`